src/TestUtils/CloudFunctionDeploymentTrait.php

```diff
@@ -39,6 +39,11 @@ trait CloudFunctionDeploymentTrait
 
     /** @var Google\Cloud\Logging\LoggingClient */
     private static $loggingClient;
+    
+    /**
+     * @var int default times to retry logs request
+     */
+    protected $eventuallyConsistentRetryCount = 5;
```

Comment:
Let's keep `3` as the default, and increase this value explicitly in the tests.

Reply:
My thought here was that if we expect logging to take more than 3 retries in general (which I believe we do?) then we could just increase the default. Another option would be to put a sleep here, and add a `$sleep` parameter for it.

Reply:
> put a sleep here, and add a $sleep parameter for it.

I'd go for that, personally.

Reply:
added!